deepsensor/model/model.py

```diff
@@ -899,10 +899,15 @@ def stitch_clipped_predictions(patch_preds, patch_overlap, patches_per_row, x1_a
                         else:
                             patch_x2 = data_array.coords[orig_x2_name].max().values, data_array.coords[orig_x2_name].min().values
                         patch_x1_index, patch_x2_index =  get_index(patch_x1, patch_x2)
-                        
+
+                        height = data_array.sizes[orig_x1_name]
+                        width = data_array.sizes[orig_x2_name]
+                        print('height and width of patch', height, width)
```

Comment:
Note to self: remove print statements in this PR

Reply:
Hopefully done now :)